File: trunk/NAI/Android/src/dk/itu/nai/comm/messages/incomming/IncomingBodyLessMessage.java
package dk.itu.nai.comm.messages.incomming;

import dk.itu.nai.comm.Protocol;

public class IncomingBodyLessMessage extends IncomingMessage {

	public IncomingBodyLessMessage(Protocol command) {
		super(command);
	}

	@Override
	protected void parseMessage(byte[] messageBody) {}

}

File: NAI/Surface/NAI/Communication/MessageLayer/Messages/Incoming/IncomingBodyLessMessage.cs

namespace NAI.Communication.MessageLayer.Messages.Incoming
{
    internal abstract class IncomingBodyLessMessage : IncomingMessage
    {
        public IncomingBodyLessMessage(MessageProtocol command)
            : base(command)
        { }

        protected override void ParseMessage(byte[] messageBody)
        { }
    }
}
